dactyl-config.yml

```diff
@@ -195,6 +195,8 @@ targets:
             # Fix links from untranslated AMM pages:
             "transaction-common-fields.html#global-flags": "transaction-common-fields.html#グローバルフラグ"
             "currency-formats.html#nonstandard-currency-codes": "currency-formats.html#非標準通貨コード"
+            # Fix link from untranslated health-check.html:
+            "amendments.html#amendment-blocked-servers": "amendments.html#amendment-blocked"
```

Comment:
I did a double-take at the "translated" link, but [it is correct](https://xrpl.org/ja/amendments.html#amendment-blocked). 😆